packages/cli/oclif.manifest.json

```diff
@@ -1652,13 +1652,14 @@
           "allowNo": false,
           "description": "Skips the Partners Dashboard URL update step.",
           "env": "SHOPIFY_FLAG_NO_UPDATE",
-          "hidden": false,
+          "hidden": true,
```

Comment:
This is an auto-generated file- do these changes persist through a `pnpm refresh-manifests`?